mediaire_toolbox/transaction_db/transaction_db.py

```diff
@@ -125,28 +125,55 @@ def set_processing(self,
             self.session.rollback()
             raise
 
-    def set_failed(self, id_: int, cause: str):
+    def set_failed(self, id_: int, cause: str = None):
```

Comment:
Why is it now possible that a transaction fails without a cause? I thought it's better to enforce always a cause

Reply:
Ah, before I used to set skipped as failed state. Will enfore cause.